tests/integration/tier1/resolve-dependency/install_dependency_packages.py

```diff
@@ -45,6 +45,8 @@ def test_install_dependency_packages(shell):
         elif system_version.major == 8:
```

Comment:
The test is switched off for OL 8: https://github.com/oamg/convert2rhel/commit/7920fc401fd73cf191b9d298d2fe8abde1142dc3#diff-effa0515e248da3fe738adfe9c21f2977701c9e1c8701bcc86c36acb32415a43R93.

Switch it on again by removing the `adjust` clause there.